lumigator/frontend/src/components/experiments/LExperimentForm.vue

```diff
@@ -231,30 +231,50 @@ async function handleRunExperimentClicked() {
     system_prompt: experimentPrompt.value || defaultPrompt.value,
   }
 
-  const workflows = await experimentsService.createExperimentWithWorkflows(
+  const [experimentId, workflowResults] = await experimentsService.createExperimentWithWorkflows(
     experimentPayload,
     modelSelection.value.selectedModels,
   )
+  // Separate successful workflows and errors
+  const workflows = workflowResults.filter((r) => r.status === 'fulfilled').map((r) => r.value)
+  const failures = workflowResults
+    .filter((r) => r.status === 'rejected')
+    .map((r) => r.reason?.response?.data?.detail)
```

Comment:
```suggestion
  const failures = createWorkflowResponses
    .filter((promise) => promise.status === 'rejected')
    .map((promise) => promise.reason.response?.data?.detail)
```

We could technically use `getAxiosError` to capture all kinds of errors aswell including network/server down errors, not just the ones returned by the api in the body

```suggestion
  const failures = createWorkflowResponses
    .filter((promise) => promise.status === 'rejected')
    .map((promise) => {
      const errorMessage = getAxiosError(r.reason);
      try {
        const json = JSON.parse(errorMessage) // if its a json string, that means it returned `response.data`
        return json.detail
      } catch {
         return errorMessage // otherwise its just an error message / string
      }
    })
```